<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>location_view</class>
 <widget class="QWidget" name="location_view">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>491</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Adobe Arabic</family>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="text">
    <string>휴게소 위치 확인</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>360</y>
     <width>121</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>뒤로가기</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>340</y>
     <width>421</width>
     <height>51</height>
    </rect>
   </property>
   <property name="title">
    <string>좌표 검색</string>
   </property>
   <widget class="QPushButton" name="pushButton_2">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>20</y>
      <width>91</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>지도 검색하기</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>15</x>
      <y>25</y>
      <width>56</width>
      <height>12</height>
     </rect>
    </property>
    <property name="text">
     <string>휴게소 명</string>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>20</y>
      <width>201</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QWebView" name="webView">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>60</y>
     <width>551</width>
     <height>271</height>
    </rect>
   </property>
   <property name="mouseTracking">
    <bool>true</bool>
   </property>
   <property name="acceptDrops">
    <bool>true</bool>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="url">
    <url>
     <string>about:blank</string>
    </url>
   </property>
   <property name="zoomFactor">
    <double>1.000000000000000</double>
   </property>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QWebView</class>
   <extends>QWidget</extends>
   <header>QtWebKit/QWebView</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>pushButton</sender>
   <signal>clicked()</signal>
   <receiver>location_view</receiver>
   <slot>back()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>504</x>
     <y>370</y>
    </hint>
    <hint type="destinationlabel">
     <x>509</x>
     <y>343</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_2</sender>
   <signal>clicked()</signal>
   <receiver>location_view</receiver>
   <slot>local_serch()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>383</x>
     <y>372</y>
    </hint>
    <hint type="destinationlabel">
     <x>448</x>
     <y>353</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>back()</slot>
  <slot>local_serch()</slot>
 </slots>
</ui>
